tools/run_tests/run_tests_matrix.py

```diff
@@ -195,15 +195,6 @@ def _create_portability_test_jobs(extra_args=[]):
                               compiler='coreclr',
```

Comment:
Should this also be removed? I don't see C# tests in the portability matrix. This comment is for test_jobs += _generate_jobs(languages=['csharp'] on line 191 (I can't comment on it directly since it's far away from the code changed by the commit).